Makefile

```diff
@@ -140,6 +141,7 @@ docker-build:
 
 docker-bootstrap:
 	$(MAKE) docker-build
+	docker-compose up --build documentation
```

Comment:
Since openapi relies on documentation completing, we'll run them separately. Even though openapi depends on documentation, there's no guarantee that documentation finishes. See: https://docs.docker.com/compose/startup-order/